docs/docs/widgets/star-rating.md

```diff
@@ -48,7 +48,7 @@ There are currently no CSA (Component-Specific Actions) implemented to regulate
 
 | <div style={{ width:"100px"}}> Variables  </div>  | <div style={{ width:"135px"}}> Description </div> | <div style={{ width:"135px"}}> How To Access </div> |
 |:----------- |:----------- |:--------- |
-| value | This variable holds the value whenever a rating is added on the component. The variable holds a numerical value. | Access the value dynamically using JS: `{{components.colorpicker1.selectedColorHex}}`.|
+| value |Holds the value entered by the user whenever a rating is added on the component. | Accessible dynamically with JS (for e.g., `{{components.colorpicker1.selectedColorHex}}`).|
```

Comment:
Hey, can you update the query here? It should be `{{components.starrating1.value}}` instead of `{{components.colorpicker1.selectedColorHex}}`.